android/src/main/java/com/amplitude/android/utilities/ExceptionUtils.kt

```diff
@@ -0,0 +1,20 @@
+package com.amplitude.android.utilities
+
+import com.amplitude.common.Logger
+
+object ExceptionUtils {
```

Comment:
IMO, I would not pull this out into a utility class, at least without further usage. We generally want to handle all exceptions, this encourages ignoring them.

Reply:
Sounds good. That makes sense. I've removed this class